TransformationSystem/Client/TaskManager.py

```diff
@@ -333,6 +335,116 @@ def __init__( self, transClient = None, logger = None, submissionClient = None,
     self.destinationPlugin_o = None
 
   def prepareTransformationTasks( self, transBody, taskDict, owner = '', ownerGroup = '', ownerDN = '' ):
+
+    bulkFlag = self.opsH.getValue( "Transformations/BulkSubmission", False )
```

Comment:
We don't yet want to apply bulk submission to all job related transformations: first, I'd just like to try out with Simulations. More "complex" tasks like Reconstruction or Stripping may wait for now.